### DISCUSSION TITLE
Update to 3.x.x  from 2.1.2 problems

### DISCUSSION THREAD

**Discussion by nickm324** [Lissy93/dashy#1654] [Help and Support]
I followed the instructions to change the port and path to conf.yml and version and after the new version builds all my stuff is gone and like it a new install.

Didn't change the path to my existing conf file on the host side, but it appears it's not reading my conf file.

Not sure what to do at this point.

**Comment by CrazyWolf13** [comment]
Can you share your full docker command?
